From aa0091e098aa8f4cccd66484c13684b3617af322 Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Wed, 15 Apr 2015 15:36:00 -0400
Subject: [PATCH 33/41] grub2ExtractTitle and extractTitle don't do the same
 thing.

So don't try to use grub2ExtractTitle in extractTitle, which actually
already knows how to do that thing right.

Resolves: rhbz#1211887

Signed-off-by: Peter Jones <pjones@redhat.com>
---
 grubby.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/grubby.c b/grubby.c
index e4358ad..5ee20e3 100644
--- a/grubby.c
+++ b/grubby.c
@@ -829,8 +829,6 @@ static int isEntryStart(struct singleLine * line,
 static char * extractTitle(struct grubConfig *cfg, struct singleLine * line) {
     /* bracketed title... let's extract it */
     char * title = NULL;
-    if (cfg->cfi == &grub2ConfigType)
-	return grub2ExtractTitle(line);
     if (line->type == LT_TITLE) {
 	char *tmp = line->elements[cfg->cfi->titlePosition].item;
 	if (cfg->cfi->titleBracketed) {
-- 
2.4.3

